src/base.jl

```diff
@@ -640,11 +697,11 @@ end
 Return a forecast for the entire time series range stored for these parameters.
 """
 function get_forecast(
-                      ::Type{T},
-                      component::Component,
-                      initial_time::Dates.DateTime,
-                      label::AbstractString,
-                      ) where T <: Forecast
+    ::Type{T},
+    component::Component,
+    initial_time::Dates.DateTime,
+    label::AbstractString,
+) where T <: IS.Forecast
```

Comment:
We made a choice to re-export Forecast as a PowerSystems type so that the user doesn't have to know this is an IS type.  I think I still prefer what we had.

Reply:
Resolved in https://github.com/NREL/PowerSystems.jl/pull/390/commits/59e59130edc3bdb963631ba666690f44811ba0b1.